### PR TITLE
Drop openshift/cluster-api in favour of machine-api-operator

### PR DESCRIPTION
**Pull request by enxebre**
Drop openshift/cluster-api in favour of machine-api-operator
Drop cluster struct references to satisfy actuator interface
needs https://github.com/openshift/machine-api-operator/pull/440
needs https://github.com/openshift/machine-api-operator/pull/441
/hold

### CODE REVIEW

File: pkg/actuators/machine/awsclient.go [hunk 1]
```diff
@@ -14,7 +12,7 @@ type AwsClientWrapper struct {
 	client awsclient.Client
 }
 
-var _ types.CloudProviderClient = &AwsClientWrapper{}
+//var _ types.CloudProviderClient = &AwsClientWrapper{}
```
Comment (bison) [inline pkg/actuators/machine/awsclient.go]: What's the deal here? Do we not implement this interface anymore? If so, we should just delete this. If not, we should un-comment it.

Comment (enxebre) [reply pkg/actuators/machine/awsclient.go]: no, we don't, since this was a left over of when we vendored the e2e tests. Removed.

**Pull request merged**